feat(perf): Add section for anomalies to summary tabs (#4708)
getsentry/sentry-docs: src/docs/product/performance/transaction-summary.mdx

![Example of Tags Tab](./tags-tab.png)

## Anomalies

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

> The timeseries graph in **Anomalies** tab shows your event throughput over time. Our anomaly detection will highlight areas that have gone above or below the expected throughput for that time of day, week etc. based on your historical data. The different colors for anomalies represent the confidence we have that that area is anomalous.

Reviewer: ```suggestion
The time series graph in the **Anomalies** tab shows your event throughput over time. Our anomaly detection highlights areas that have gone above or below the expected throughput for that time of day, week, or other period, based on your historical data. The different colors for anomalies represent the level of confidence we have that that area is anomalous.
```

> The time series graph in the **Anomalies** tab shows your event throughput over time. Our anomaly detection highlights areas that have gone above or below the expected throughput for that time of day, week, or other period, based on your historical data. The different colors for anomalies represent the level of confidence we have that that area is anomalous.

Reviewer 2: should we say "transaction throughput" rather than "event throughput" here for the time being?

Author: I'm either/or on this usually since our users might not really conceive of 'transactions' as events, whereas events and epm always applies. We actually mix these a lot in the product, so I don't know that we've figured out which we prefer either.

Reviewer 2: fair enough!

> The time series graph in the **Anomalies** tab shows your event throughput over time. Our anomaly detection highlights areas that have gone above or below the expected throughput for that time of day, week, or other period, based on your historical data. The different colors for anomalies represent the level of confidence we have that that area is anomalous.

Reviewer 2: ```suggestion
The time series graph in the **Anomalies** tab shows your transaction throughput over time. Our anomaly detection highlights areas that have gone above or below the expected throughput for that time of day, week, or other period, based on your historical data. The shading indicates our confidence that a particular time interval is anomalous.
```
